### DISCUSSION TITLE
延时加载 功能 支不支持单个列延迟加载？

### DISCUSSION THREAD

**Discussion by sgf** [dotnetcore/FreeSql#1509] [Q&A] [answered]
延时加载 功能 支不支持单个列延迟加载？
需求是这样的：

某个表某个列是大对象（二进制数据）。
如果select一次性都加载出来的话，会消耗大量内存。

但是因此这种需求就需要在筛选的时候只加载 筛选和信息字段，但是不加载二进制数据。
当选中访问二进制字段后就需要 从数据库加载该部分数据。

**Comment by 2881099** [comment, marked as answer]
正常这种应该纵向分表。

例如 user、user_ext 两个表，user 创建 user_ext 对应的导航属性，即可实现延时加载。
